Sources/Bugsnag/PayloadTransformer.swift

```diff
@@ -93,4 +102,20 @@ internal struct PayloadTransformer: PayloadTransformerType {
     private func optionalNode(_ node: Node?) -> Node {
         return node ?? Node.null
     }
+
+    private func filterOutKeys(_ keys: [String], inNode node: Node) -> Node {
+        var outcome: [String: Node] = [:]
+
+        guard let nodeObjects = node.nodeObject else {
+            return node
+        }
+
+        for obj in nodeObjects {
```

Comment:
This function (`filterOutKeys`) looks pretty good. It could be shorter using `filter` if you are up for looking into that.